src/config.rs

```diff
@@ -322,6 +334,15 @@ mod tests {
 		serde_yaml::from_str(EXAMPLE_CONFIG).expect("invalid config")
 	}
 
+	fn create_config_file(dir: &Path) -> PathBuf {
+		let file_path = dir.join("config.yaml");
+		let mut config_file = File::create(&file_path).expect("failed to create config file");
+		config_file
+			.write_all(EXAMPLE_CONFIG.as_bytes())
+			.expect("Failed to write config file content");
+		file_path
+	}
```

Comment:
I think with this new approach we can just store `EXAMPLE_CONFIG` as a file somewhere in the repo, to avoid creating it in tempdir

Reply:
Ah, I see you compare it with static `Config`, ok, that makes sense then